devicetypes/smartthings/zigbee-thermostat.src/zigbee-thermostat.groovy

```diff
@@ -264,26 +304,53 @@ def getPowerSource(value) {
 	return result
 }
 
+def switchMode() {
+	def currentMode = device.currentValue("thermostatMode")
+	def modeOrder = state.supportedThermostatModes
+log.debug "switchMode() $currentMode $modeOrder"
```

Comment:
nit: indentation on the log statements

Reply:
Yeah, these will go away shortly. The PR isn't 100% done yet.

Actually, Bob shared some code for changes made to multi-attr tiles some time ago that should make this function irrelevant. :smile:

Reply:
With some updated tile stuff from @bflorian this function is no longer needed, so it is gone!